bindings/nodejs/examples/how_tos/outputs/output_features.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    Client,
+    initLogger,
+    Utils,
+    UnlockCondition,
+    AddressUnlockCondition,
+    MetadataFeature,
+    SenderFeature,
+    TagFeature,
+    Ed25519Address,
+    IssuerFeature,
+    utf8ToHex,
+} from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/outputs/output_features.ts
+
+// Build ouputs with all features
```

Comment:
Do you think it's better to have one output for each feature instead of all features in a single output?

Reply:
Wdym? I have one NFT output for every feature.

Reply:
Yeah, my question is if that's better than a single NFT output with all features

Reply:
Ahh. I See. Yeah not sure about that 🤔 Does it matter? I just kept it that way because it's basically a copy of the build_basic_output.

But I guess it's also nicer for later splitting the tutorial up with our tool 🤔